examples/jssp/problem/probe.rs

```diff
@@ -115,27 +115,24 @@ impl Probe<JsspIndividual> for JsspProbe {
         metadata: &ecrs::ga::GAMetadata,
         _population: &[JsspIndividual],
         best_individual: &JsspIndividual,
-    ) { 
+    ) {
         let mut ops = best_individual.operations.clone();
         ops.sort_unstable_by(|a, b| {
-            if a.finish_time == b.finish_time {
-                if a.duration == 0 && b.duration != 0 {
-                    return Ordering::Greater;
-                } else if a.duration != 0 && b.duration == 0 {
-                    return Ordering::Less;
-                } else if a.machine < b.machine {
-                    return Ordering::Less;
-                } else {
-                    return Ordering::Greater;
-                }
-            } else if a.finish_time < b.finish_time {
-                return Ordering::Less;
+            if a.finish_time < b.finish_time {
+                Ordering::Less
+            } else if a.finish_time > b.finish_time {
+                Ordering::Greater
+            } else if a.duration != 0 && b.duration != 0 {
+               a.machine.cmp(&b.machine)
+            } else if a.duration != 0 && b.duration == 0 {
+                Ordering::Less
             } else {
-                return Ordering::Greater;
+                Ordering::Greater
             }
         });
```

Comment:
Not sure what should happen when:

1. finish times are equal and both durations are 0
2. finish times are equal and both durations are not equal and not 0